vtex/andromedocs
# Feature/add sidebar

#### What is the purpose of this pull request?
Add a Sidebar to layout of theme

#### What problem is this solving?
Resolve #28 

#### How should this be manually tested?
Sidebar.yaml already exists with an example menu, but you can change it and see the changes in the Sidebar menu

run (in the /andromedocs-theme folder)
> gatsby develop 
    
or
> yarn workspace andromedocs-theme develop 


#### Types of changes

#### How does this PR make you feel? [:link:](http://giphy.com/)

<img src="https://media.tenor.com/images/5cb9646ba33e330ac4bb750fc89b4f35/tenor.gif" alt="happy" width="400"/>
